.eslintrc.json

```
@@ -8,9 +8,49 @@
     "es6": true
   },
   "plugins": [
-    "office-addins"
+    "office-addins",
+    "@typescript-eslint",
+    "unused-imports"
   ],
   "extends": [
-    "plugin:office-addins/react"
-  ]
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:office-addins/react",
+    "airbnb",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:prettier/recommended"
+    ],
+  "parser": "@typescript-eslint/parser",
```

Perhaps also

```suggestion
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    }
  }
```

although it doesn't seem to have caused any issues having it off?